ERROR: relation "resource_events_status_for_corrective_change_idx" already exists

git clone and running docker-compose up, without any changes.

error returned on deployment, 
at 'Applying database migration version 65':

<code>
2019-06-24 12:08:12,991 INFO  [p.p.s.migrate] Applying database migration version 64
2019-06-24 12:08:12,992 INFO  [p.p.s.migrate] [1/8] Cleaning up unreferenced facts...
2019-06-24 12:08:12,993 INFO  [p.p.s.migrate] [2/8] Creating new fact storage tables...
2019-06-24 12:08:13,001 INFO  [p.p.s.migrate] [3/8] Copying unique fact values into fact_values
2019-06-24 12:08:13,002 INFO  [p.p.s.migrate] [4/8] Reconstructing facts to refer to fact_values...
2019-06-24 12:08:13,004 INFO  [p.p.s.migrate] [5/8] Cleaning up duplicate null values...
2019-06-24 12:08:13,006 INFO  [p.p.s.migrate] [6/8] Computing fact value hashes...
2019-06-24 12:08:13,007 INFO  [p.p.s.migrate] [7/8] Indexing fact_values table...
2019-06-24 12:08:13,017 INFO  [p.p.s.migrate] [8/8] Indexing facts table...
2019-06-24 12:08:13,028 INFO  [p.p.s.migrate] Applied database migration version 64 in 37 ms
2019-06-24 12:08:13,028 INFO  [p.p.s.migrate] Applying database migration version 65
2019-06-24 12:08:13,046 ERROR [p.p.s.migrate] Caught SQLException during migration
java.sql.BatchUpdateException: Batch entry 1 alter table resource_events
     alter column status type text,
     alter column property type text,
     alter column containing_class type text,
     alter column file type text was aborted: ERROR: relation "resource_events_status_for_corrective_change_idx" already exists  Call getNextException to see other errors in the batch.
	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:148)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2179)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:479)
	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:835)
	at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
	at com.zaxxer.hikari.pool.HikariProxyStatement.executeBatch(HikariProxyStatement.java)
	at clojure.java.jdbc$execute_batch.invokeStatic(jdbc.clj:563)
	at clojure.java.jdbc$execute_batch.invoke(jdbc.clj:556)
	at clojure.java.jdbc$db_do_commands$fn__21283.invoke(jdbc.clj:906)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:814)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:741)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:754)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:741)
	at clojure.java.jdbc$db_do_commands.invokeStatic(jdbc.clj:905)
	at clojure.java.jdbc$db_do_commands.invoke(jdbc.clj:890)
	at puppetlabs.puppetdb.jdbc$do_commands.invokeStatic(jdbc.clj:38)
	at puppetlabs.puppetdb.jdbc$do_commands.doInvoke(jdbc.clj:33)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at puppetlabs.puppetdb.scf.migrate$varchar_columns_to_text.invokeStatic(migrate.clj:1467)
	at puppetlabs.puppetdb.scf.migrate$varchar_columns_to_text.invoke(migrate.clj:1466)
	at puppetlabs.puppetdb.scf.migrate$migrate_BANG_$fn__34737$fn__34739$fn__34743.invoke(migrate.clj:1793)
	at puppetlabs.puppetdb.scf.migrate$sql_or_die.invokeStatic(migrate.clj:1735)
	at puppetlabs.puppetdb.scf.migrate$sql_or_die.invoke(migrate.clj:1726)
	at puppetlabs.puppetdb.scf.migrate$migrate_BANG_$fn__34737$fn__34739.invoke(migrate.clj:1793)
	at clojure.core$map$fn__5851.invoke(core.clj:2755)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clojure.core$filter$fn__5878.invoke(core.clj:2809)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:58)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clojure.core$map$fn__5851.invoke(core.clj:2746)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.boundedLength(RT.java:1788)
	at clojure.lang.RestFn.applyTo(RestFn.java:130)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at puppetlabs.puppetdb.scf.migrate$migrate_BANG_$fn__34737.invoke(migrate.clj:1800)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:771)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:741)
	at puppetlabs.puppetdb.scf.migrate$migrate_BANG_.invokeStatic(migrate.clj:1788)
	at puppetlabs.puppetdb.scf.migrate$migrate_BANG_.invoke(migrate.clj:1759)
	at puppetlabs.puppetdb.cli.services$initialize_schema.invokeStatic(services.clj:331)
	at puppetlabs.puppetdb.cli.services$initialize_schema.invoke(services.clj:316)
	at puppetlabs.puppetdb.cli.services$init_with_db$fn__34979.invoke(services.clj:363)
	at puppetlabs.puppetdb.cli.services$init_with_db.invokeStatic(services.clj:362)
	at puppetlabs.puppetdb.cli.services$init_with_db.invoke(services.clj:335)
	at puppetlabs.puppetdb.cli.services$start_puppetdb.invokeStatic(services.clj:458)
	at puppetlabs.puppetdb.cli.services$start_puppetdb.invoke(services.clj:432)
	at puppetlabs.puppetdb.cli.services$reify__35099$service_fnk__18770__auto___positional$reify__35110.start(services.clj:568)
	at puppetlabs.trapperkeeper.services$fn__18607$G__18588__18610.invoke(services.clj:9)
	at puppetlabs.trapperkeeper.services$fn__18607$G__18587__18614.invoke(services.clj:9)
	at puppetlabs.trapperkeeper.internal$fn__19102$run_lifecycle_fn_BANG___19109$fn__19110.invoke(internal.clj:198)
	at puppetlabs.trapperkeeper.internal$fn__19102$run_lifecycle_fn_BANG___19109.invoke(internal.clj:181)
	at puppetlabs.trapperkeeper.internal$fn__19131$run_lifecycle_fns__19136$fn__19137.invoke(internal.clj:231)
	at puppetlabs.trapperkeeper.internal$fn__19131$run_lifecycle_fns__19136.invoke(internal.clj:208)
	at puppetlabs.trapperkeeper.internal$fn__19704$build_app_STAR___19713$fn$reify__19725.start(internal.clj:586)
	at puppetlabs.trapperkeeper.internal$fn__19752$boot_services_for_app_STAR__STAR___19759$fn__19760$fn__19762.invoke(internal.clj:612)
	at puppetlabs.trapperkeeper.internal$fn__19752$boot_services_for_app_STAR__STAR___19759$fn__19760.invoke(internal.clj:610)
	at puppetlabs.trapperkeeper.internal$fn__19752$boot_services_for_app_STAR__STAR___19759.invoke(internal.clj:604)
	at clojure.core$partial$fn__5826.invoke(core.clj:2630)
	at puppetlabs.trapperkeeper.internal$fn__19176$initialize_lifecycle_worker__19187$fn__19188$fn__19338$state_machine__12279__auto____19363$fn__19366.invoke(internal.clj:251)
	at puppetlabs.trapperkeeper.internal$fn__19176$initialize_lifecycle_worker__19187$fn__19188$fn__19338$state_machine__12279__auto____19363.invoke(internal.clj:251)
	at clojure.core.async.impl.ioc_macros$run_state_machine.invokeStatic(ioc_macros.clj:973)
	at clojure.core.async.impl.ioc_macros$run_state_machine.invoke(ioc_macros.clj:972)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invokeStatic(ioc_macros.clj:977)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invoke(ioc_macros.clj:975)
	at clojure.core.async$ioc_alts_BANG_$fn__12494.invoke(async.clj:384)
	at clojure.core.async$do_alts$fn__12434$fn__12437.invoke(async.clj:253)
	at clojure.core.async.impl.channels.ManyToManyChannel$fn__7098$fn__7099.invoke(channels.clj:95)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "resource_events_status_for_corrective_change_idx" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)
	... 77 common frames omitted

</code>

running provided RSpec as described also returns failures:

Failures:

  1) The docker-compose file works the cluster starts should start puppetdb
     Failure/Error: expect(wait_on_puppetdb_status()).to eq('running')
     
     Timeout::Error:
       puppetdb never entered running state


Amir.

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -28,7 +28,7 @@ services:
          - puppet.${DOMAIN:-internal}
 
   postgres:
-    image: postgres:9.6.13
+    image: postgres:9.6
     environment:
       - POSTGRES_PASSWORD=puppetdb
       - POSTGRES_USER=puppetdb

